### PR TITLE
Add number of results in the end of the process

### PR DESCRIPTION
**Pull request by ediamantopoulou**
I have changed the notify.py in order to show: The number of results when the process is finished.

### CODE REVIEW

File: sherlock/tests/test_multiple_usernames.py [hunk 1]
```diff
@@ -7,7 +7,19 @@
 checksymbols = []
 checksymbols = ["_", "-", "."]
 
-class TestMulripleUsernames(unittest.TestCase):
+"""Test for mulriple usernames.
```
Comment (MoMMde) [inline sherlock/tests/test_multiple_usernames.py:10]: looks like a typo

**Review by TheYahya — APPROVED** [review]

File: sherlock/notify.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 """
 from result import QueryStatus
 from colorama import Fore, Style
-
+globvar = 0 # global variable to count the number of results.
```
Comment (ediamantopoulou) [inline sherlock/notify.py:8]: add a global variable "globvar" to be visible to the whole class and count the number of social medias.

File: sherlock/notify.py [hunk 3]
```diff
@@ -185,6 +184,20 @@ def finish(self, message="The processing has been finished."):
...
+        Keyword Arguments:
+        self                   -- This object.
+
+        Return Value:
+        The number of results by the time we call the function.
+        """
+        global globvar
+        globvar += 1
+        return globvar
+
```
Comment (ediamantopoulou) [inline sherlock/notify.py:200]: for every social media in which the username exists, the "globvar" parameter is increasing.

File: sherlock/notify.py [hunk 4]
```diff
@@ -206,14 +219,15 @@ def update(self, result):
         
         # Output to the terminal is desired.
         if result.status == QueryStatus.CLAIMED:
+            self.countResults()
```
Comment (ediamantopoulou) [inline sherlock/notify.py:222]: for every social media in which the username exists, the function countResults() is called

**Pull request merged**